.github/workflows/pipeline.yml

```diff
@@ -75,8 +79,6 @@ jobs:
   deploy_prod:
     name: Deploy to the prod environment
     needs:
-      - build
```

Comment:
don't really understand why these were here - they are pre-requisites for `deploy_dev` anyway so don't need to be repeated here